var/spack/repos/builtin/packages/fox/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Fox(AutotoolsPackage):
+    """FOX is a C++ based Toolkit for developing Graphical User Interfaces
+    easily and effectively. It offers a wide, and growing, collection of
+    Controls, and provides state of the art facilities such as drag and drop,
+    selection, as well as OpenGL widgets for 3D graphical manipulation. FOX
+    also implements icons, images, and user-convenience features such as status
+    line help, and tooltips. Tooltips may even be used for 3D objects!"""
+
+    homepage = "http://fox-toolkit.org/"
+    url      = "http://fox-toolkit.org/ftp/fox-1.7.67.tar.gz"
+
+    version('1.7.67', sha256='7e511685119ef096fa90d334da46f0e50cfed8d414df32d80a7850442052f57d')
+    version('1.6.57', preferred=True, sha256='65ef15de9e0f3a396dc36d9ea29c158b78fad47f7184780357b929c94d458923')
```

Comment:
I would make this not preferred, see below

Reply:
Well, my intention was to set the version constraint as per the comment below but I somehow neglected to do that. I included version 1.7.65 because there was a substantial time gap between releases. The 1.7.65 is currently tagged as "development" but I thought it might be useful to have. With that, I think setting 1.6.57, tagged as 'stable', should be set as preferred.